### PR TITLE
Release rospilot 1.6.0 for Noetic

### PR DESCRIPTION
**Pull request by cberner**

### CODE REVIEW

File: noetic/distribution.yaml [hunk 1]
```diff
@@ -3961,6 +3961,17 @@ repositories:
       type: git
       url: https://github.com/PickNikRobotics/rosparam_shortcuts.git
       version: noetic-devel
+  rospilot:
+    release:
```
Comment (hidmic) [inline noetic/distribution.yaml:3965]: @cberner no `doc` entry? Just checking.

**Review by hidmic — COMMENTED** [review]
@cberner I see you bloomed this new version. Why not letting `bloom` open the PR for you?

**Comment by cberner** [comment]
Bloom failed to open the PR with an error, and said to do it manually

File: noetic/distribution.yaml (hunk 1 above)
Comment (cberner) [reply noetic/distribution.yaml:3965]: Correct, none at the moment

**Comment by hidmic** [comment]
> Bloom failed to open the PR with an error, and said to do it manually

Any chance you can update the PR title and description to match that of `bloom`. For clean history's sake.

**Comment by cberner** [comment]
Closing in favor of https://github.com/ros/rosdistro/pull/26485 I figured out how to fix my bloom errors

**Pull request closed without merging**